rust/src/quic/frames.rs

```diff
@@ -115,13 +123,210 @@ impl fmt::Display for StreamTag {
     }
 }
 
+#[derive(Debug, PartialEq)]
+pub(crate) struct Ack {
+    pub largest_acknowledged: u64,
+    pub ack_delay: u64,
+    pub ack_range_count: u64,
+    pub first_ack_range: u64,
+}
+
+#[derive(Debug, PartialEq)]
+pub(crate) struct Crypto {
+    pub ciphers: Vec<TlsCipherSuiteID>,
+    pub extv: Vec<QuicTlsExtension>,
+    //does not work, because of lifetime due to references to the slice used for parsing
```

Comment:
can this comment move to `extv` just above and explain why it is like this instead of using `Vec<TlsExtension>`? Comments about why code that is in use are like they are, are more useful than comments explaining commented out things, I think.

Reply:
ok